DigitalOffice/Tests/CheckRightsServiceTests/Controllers/CheckRightsControllerTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+using System.Collections.Generic;
+using LT.DigitalOffice.CheckRightsService.Commands.Interfaces;
+using LT.DigitalOffice.CheckRightsService.Controllers;
+using LT.DigitalOffice.CheckRightsService.Models;
+using Moq;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.CheckRightsServiceUnitTests.Controllers
+{
+    public class CheckRightsControllerTests
+    {
+        private CheckRightsController checkRightsController;
+        private Mock<IGetRightsListCommand> commandMock;
+        private Right right;
+
+        [SetUp]
+        public void SetUp()
+        {
+            checkRightsController = new CheckRightsController();
+            commandMock = new Mock<IGetRightsListCommand>();
+            right = new Right {Id = 0, Name = "Right", Description = "Allows you everything"};
+        }
+
+        [Test]
+        public void ShouldReturnTheSameValueAsCommandWhenGettingRightsList()
+        {
+            commandMock.Setup(command => command.Execute())
+                .Returns(new List<Right> {right})
+                .Verifiable();
+
+            Assert.That(checkRightsController.GetRightsList(commandMock.Object),
+                Is.EquivalentTo(new List<Right> {right}));
+            commandMock.Verify();
+        }
+
+        [Test]
+        public void ShouldThrowExceptionWhenCommandThrowsException()
```

Comment:
You can clarify the description of the method.

Reply:
It can also be helpful to separate these tests by the appropriate region.

Reply:
If the need arises, then the person adding new tests will be able to rename